src/useClickedOutside/useClickedOutside.ts

```diff
@@ -0,0 +1,41 @@
+import type { RefElementType } from '@muban/muban';
+import { useEventListener } from '../useEventListener/useEventListener';
+import type { DomElementOrRef, DomElementsOrRefCollection } from '../utils/util.types';
+import { getElements } from '../utils/getElements';
+
+/**
+ * A small wrapper method to easily check if a user clicked inside a HTMLElement.
+ *
+ * @param target - The target there the user clicked.
+ * @param container - The container where to check against.
+ */
+export function isClickedInside(target: EventTarget, container: RefElementType): boolean {
+  return container ? container.contains(target as HTMLElement) : false;
+}
+
+/**
+ * Will add an event listener to the body and will fire a callback if the user clicks
```

Comment:
`the body` ? line 40 suggests that the event listener is attached to the whole `document`, not the `body`